### PR TITLE
filters util function

### PR DESCRIPTION
**Pull request by rotty3000**
Signed-off-by: Raymond Augé <raymond.auge@liferay.com>

### CODE REVIEW

File: org.osgi.test.common/pom.xml [hunk 1]
```diff
@@ -31,6 +31,13 @@
 		OSGi Test Common Support
 	</description>
 
+	<dependencies>
+		<dependency>
+			<groupId>org.osgi</groupId>
+			<artifactId>osgi.core</artifactId>
```
Comment (bjhargrave) [inline org.osgi.test.common/pom.xml]: If our intent is to make proper bundles of these jars, we need to make sure we use the lowest framework version possible to accomplish our goals. So using Core R7 means no Core R6 users can use our bundles.

File: org.osgi.test.common/src/main/java/org/osgi/test/common/Filters.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	 *
+	 * @param format a format string
+	 * @param args format arguments
+	 * @return filter
+	 */
+	public static Filter format(String format, Object... args) {
+		try {
+			return FrameworkUtil.createFilter(String.format(format, args));
+		} catch (InvalidSyntaxException ise) {
+			throw new AssertionError("Invalid filter", ise);
```
Comment (bjhargrave) [inline org.osgi.test.common/src/main/java/org/osgi/test/common/Filters.java]: Shouldn't this be one of the Junit assertion failure types? Or just duck the exception type and rethrow.

File: org.osgi.test.common/pom.xml (hunk 1 above)
Comment (rotty3000) [reply org.osgi.test.common/pom.xml]: good point, reducing to 6

File: org.osgi.test.common/src/main/java/org/osgi/test/common/Filters.java (hunk 2 above)
Comment (rotty3000) [reply org.osgi.test.common/src/main/java/org/osgi/test/common/Filters.java]: throwing ducks

**Pull request merged**